Packages/vcs/Lib/VTKPlots.py

```diff
@@ -1076,9 +1077,11 @@ def png(self, file, width=None, height=None,
         if width is not None and height is not None:
             if self.renWin.GetSize() != (width, height):
                 user_dims = (self.canvas.bgX, self.canvas.bgY, sz[0], sz[1])
-                self.renWin.SetSize(width, height)
+                # We need to set canvas.bgX and canvas.bgY before we do renWin.SetSize
+                # otherwise, canvas.bgX,canvas.bgY will win
```

Comment:
> Because you will endup, in certain cases, with bgX, bgY, not with the parameters you pass in SetSize

if SetSize triggers something which set bgX and bgY in some cases which won't be same as the origin args then I think we have problem. I think its worth looking into it. Thoughts?